src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -30,15 +30,15 @@ const ModalWindow: FC = () => {
     }
   };
 
-  const handleChangeMessage = (e: React.ChangeEvent<HTMLInputElement>): void => {
+  const handleChangeMessage = (e: ChangeEvent<HTMLInputElement>): void => {
```

Comment:
You can try to join 3 functions in 1
Pass second parameter as fieldName: string
Then setInput({...input, [fieldName]: e.target.value})

Reply:
Yes, I try, but I don't like how onChange look like inside ' onChange={(e: any) => handleChange(e, "message")} '  , maybe somehow it can be optimized

Reply:
Join 3 functions in 1